### PR TITLE
kapacitor 1.6.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
# pkg-config --cflags  -- flux
Started pkg-config	{"arg0": "/private/tmp/kapacitor-20210628-11825-1bfn39y/bootstrap/pkg-config", "args": ["--cflags", "--", "flux"]}
Found pkg-config executable	{"path": "/opt/homebrew/Library/Homebrew/shims/mac/super/pkg-config"}
Determined module root	{"path": "/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/influxdata/flux@v0.116.1-0.20210519190248-4347b978c91a"}
Flux module is the main module	{"modroot": "/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/influxdata/flux@v0.116.1-0.20210519190248-4347b978c91a"}
Unable to determine cargo target. Using the default.	{"target": "darwin_arm64"}
Executing cargo build	{"dir": "/Users/brew/Library/Caches/Homebrew/go_cache/pkgconfig/github.com/influxdata/flux@v0.116.1-0.20210519190248-4347b978c91a/libflux", "target": ""}
Error installing library	{"name": "flux", "error": "exec: \"cargo\": executable file not found in $PATH"}
pkg-config: exit status 1
```

File: Formula/kapacitor.rb [hunk 1]
```diff
@@ -20,8 +20,21 @@ class Kapacitor < Formula
   end
 
   depends_on "go" => :build
+  depends_on "rust" => :build
+
+  # NOTE: The version here is specified in the go.mod of influxdb.
```
Comment (carlocab) [inline Formula/kapacitor.rb:25]: Is this comment right? It refers to `influxdb`. Do we need to check that `go.mod` too? Or is it in `kapacitor`'s `go.mod`?

Comment (chenrui333) [reply Formula/kapacitor.rb:25]: good catch, fixing the comment reference now. :)

Comment (chenrui333) [reply Formula/kapacitor.rb]: Thanks!

**Review by SMillerDev — APPROVED** [review]
Can't the pkg-config wrapper be a formula?

**Pull request closed without merging**